agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/ConfigurationBuilder.java

```diff
@@ -191,7 +191,13 @@ private static void logConfigurationWarnings(Configuration config) {
               + " and it is now enabled by default,"
               + " so no need to enable it under preview configuration");
     }
-    for (SamplingOverride override : config.preview.sampling.overrides) {
+
+    if (!config.preview.sampling.overrides.isEmpty()) {
+      configurationLogger.warn(
+          "\"Sampling overrides\" is no longer in preview and it has been GA since 3.5 GA,");
+      config.sampling.overrides = config.preview.sampling.overrides;
+    }
+    for (SamplingOverride override : config.sampling.overrides) {
       if (override.telemetryKind != null) {
```

Comment:
@trask probably can drop supported deprecated 'sampling overrides' attributes at this point?